databricks/koalas/internal.py

```diff
@@ -63,6 +63,8 @@
 
 HIDDEN_COLUMNS = {NATURAL_ORDER_COLUMN_NAME}
 
+SERIES_DEFAULT_NAME = "0"
```

Comment:
How about `SPARK_DEFAULT_SERIES_NAME`?

Reply:
@ueshin 

Hmm, It looks better in terms of maintaining consistency with `SPARK_DEFAULT_INDEX_NAME`.

Thanks!